src/website/app/demos/Button/index.js

```diff
@@ -26,6 +26,7 @@ export default {
   doc,
   examples,
   slug: 'button',
+  syntax: `import Button from 'mineral-ui/Button'`,
```

Comment:
I think `importSyntax` would be more clear, though the property's value makes it immediately obvious, so 🤷‍♂️.

Reply:
I'd suggest `importSyntax` as the key as well.

Reply:
I had that first, but I'm a sucker for brevity. `importSyntax` is probably better in the long run.